independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java

```diff
@@ -60,26 +60,14 @@ public VertxResteasyReactiveRequestContext(Deployment deployment, ProvidersImpl
         this.devModeTccl = devModeTccl;
         context.addHeadersEndHandler(this);
         String expect = request.getHeader(HttpHeaderNames.EXPECT);
-        ContextInternal internal = ((ConnectionBase) context.request().connection()).getContext();
-        if (!vertxContextPropsToCopy.isEmpty()) {
-            ContextInternal current = (ContextInternal) Vertx.currentContext();
-            Map<Object, Object> internalLocalContextData = internal.localContextData();
-            Map<Object, Object> currentLocalContextData = current.localContextData();
-            for (int i = 0; i < vertxContextPropsToCopy.size(); i++) {
-                String name = vertxContextPropsToCopy.get(i);
-                Object value = currentLocalContextData.get(name);
-                if (value != null) {
-                    internalLocalContextData.put(name, value);
-                }
-            }
-        }
```

Comment:
There was a reason @radcortez had done this copying of local context data IIRC. Roberto, do you remember? If so, how was it tested?

Reply:
I think the original premise of using the event loops context was wrong (((ConnectionBase) context.request().connection()).getContext()), the copy would kind of work around that by copying stuff from the duplicated context into the original context, but really we should always be using the duplicated context.

Reply:
Yeah, I agree. I just want to make sure whatever issue was being fixed by Roberto's change is not broken by this.

Reply:
Damned! I didn't see the fact it was retrieving the connection context and not the duplicated context!

Reply:
TooManyContextsException!

Reply:
Interesting, I've had this conversation with @cescoffier before:
https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/VertxResteasyReactiveRequestContext/near/266843412

I did suggest using the duplicated context (because it contained the OpenTelemetry context to propagate).

Reply:
It would seem the `CopyVertxContextDataBuildItem` is no longer applied after this patch. Is that OK

In case it's intentional, feel free to un-draft:
 - https://github.com/quarkusio/quarkus/pull/23424